src/main/resources/org/jenkinsci/plugins/workflow/cps/samples/github-maven.groovy

```diff
@@ -0,0 +1,32 @@
+pipeline {
+   agent any
+
+   tools {
+      // Install the Maven version configured as "M3" and add it to the path.
+      maven "M3"
+   }
+
+   stages {
+      stage('Build') {
+         steps {
+            // Get some code from a GitHub repository
+            git 'https://github.com/jglick/simple-maven-project-with-tests.git'
```

Comment:
what about something from here https://github.com/cloudbeers?

Reply:
For this PR my goal was simply to introduce the extension point. Content changes can be done as follow-ups (and more easily reviewed as real `*.groovy` files).